pkg/net/libp2p/key.go

```diff
@@ -27,15 +27,32 @@ func operatorPrivateKeyToNetworkKeyPair(operatorPrivateKey *operator.PrivateKey)
 		return nil, nil, fmt.Errorf("libp2p supports only secp256k1 operator keys")
 	}
 
-	// Libp2p keys are actually btcec keys under the hood.
-	btcecPrivateKey, btcecPublicKey := btcec.PrivKeyFromBytes(
-		DefaultCurve, operatorPrivateKey.D.Bytes(),
+	// Note that `libp2pcrypto.UnmarshalSecp256k1PrivateKey` uses the secp256k1
+	// implementation provided by decred underneath
+	privKey, err := libp2pcrypto.UnmarshalSecp256k1PrivateKey(
```

Comment:
Let's be specific about the key name and avoid abbreviations. I propose naming it `libp2pPrivateKey`

Reply:
Done in 86d4e21916f151b85db250a1431977fe4ca6ab8a.